src/range-slider/styles/range-slider.m.css

```diff
@@ -18,6 +17,8 @@
 /* Make native input invisible */
 .nativeInput {
 	appearance: none;
+	-moz-appearance: none;
+	-webkit-appearance: none;
```

Comment:
Do we not have auto-prefixing?

Reply:
We don't at the moment for the examples application because it runs in the "speed" mode that disables postcss. This is mostly temporary until we add auto prefixing as part of build-app under speed mode. I will raise an issue on both repos to ensure that it is followed.